### PR TITLE
Add initial manifest list create tests

### PR DESCRIPTION
**Pull request by tophj-ibm**
Signed-off-by: Christopher Jones <tophj@linux.vnet.ibm.com>

### CODE REVIEW

File: cli/command/manifest/create_test.go [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+	cmd := newCreateListCommand(cli)
+	cmd.SetArgs([]string{"example.com/list:v1", "example.com/alpine:3.1"})
+	cmd.Flags().Set("amend", "true")
+	cmd.SetOutput(ioutil.Discard)
+	err = cmd.Execute()
+	require.NoError(t, err)
+
+	// make a new cli to clear the buffers
+	cli = test.NewFakeCli(nil)
+        cli.SetManifestStore(store)
```
Comment (clnperez) [inline cli/command/manifest/create_test.go]: space

File: cli/command/manifest/annotate.go [hunk 2]
```diff
@@ -47,11 +47,11 @@ func newAnnotateCommand(dockerCli command.Cli) *cobra.Command {
 func runManifestAnnotate(dockerCli command.Cli, opts annotateOptions) error {
 	targetRef, err := normalizeReference(opts.target)
 	if err != nil {
-		return errors.Wrapf(err, "annotate: Error parsing name for manifest list (%s): %s", opts.target)
+		return errors.Wrapf(err, "annotate: Error parsing name for manifest list %s", opts.target)
```
Comment (clnperez) [inline cli/command/manifest/annotate.go]: case

File: cli/command/manifest/create_test.go [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+// attempt to make a manifest list without valid images
+func TestManifestCreateNoManifest(t *testing.T) {
+	store, cleanup := newTempManifestStore(t)
+	defer cleanup()
+
+	cli := test.NewFakeCli(nil)
+	cli.SetManifestStore(store)
+	cli.SetRegistryClient(&fakeRegistryClient{
+		getManifestFunc: func(_ context.Context, ref reference.Named) (manifesttypes.ImageManifest, error) {
+			return manifesttypes.ImageManifest{}, errors.Errorf("No such image: %v", ref)
```
Comment (clnperez) [inline cli/command/manifest/create_test.go:106]: 1) try without overriding getManifestFunc
2) update error elsewhere to make more informative

**Review by clnperez — CHANGES_REQUESTED** [review]

File: cli/command/manifest/create_test.go (hunk 3 above)
Comment (tophj-ibm) [reply cli/command/manifest/create_test.go:106]: Need this here. Fixed everything else

**Pull request merged**